uber/templates/preregistration/index.html

```diff
@@ -97,10 +82,34 @@
         <th></th>
     </tr>
 </tfoot>
-
-</table>
 {% endif %}
 
+</table></div>
+
+<div class="row">
+<div class="row">
+    <div class="text-center">
+        {% if AT_THE_CON %}
+        <div class="bg-danger" style="margin: 20px; padding: 10px;"><strong>NOTE: A government-issued ID is required to pick up your badge.
```

Comment:
This warning was an AC-specific request. The other stuff is part of the pre-reg/at-door merge.